docs/guides/local-testing-mode.ipynb

```diff
@@ -37,12 +39,36 @@
+    "- **Aer simulator**: Simulators from [Qiskit Aer](/docs/guides/simulate-with-qiskit-aer) provide higher-performance simulation that can handle larger circuits and [custom noise models](/docs/guides/build-noise-models). A list of simulation method options are available when you use `AerSimulator` in local testing mode. See the [Clifford simulation mode example](#clifford-sim), which demonstrates how to efficiently simulate Clifford circuits with a large number of qubits.\n",
+    "\n",
+    "    <details>\n",
+    "    <summary>**List of simulation methods available from Qiskit Aer**</summary>\n",
+    "\n",
+    "    See the [`AerSimulator`](https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator) documentation for more information.\n",
+    "\n",
+    "    * ``\"automatic\"``: Default simulation method. Select the simulation method automatically, based on the circuit and noise model.\n",
+    "\n",
+    "    * ``\"statevector\"``: A dense statevector simulation that can sample measurement outcomes from *ideal* circuits with all measurements at end of the circuit. For noisy simulations, each shot samples a randomly sampled noisy circuit from the noise model.\n",
+    "\n",
+    "    * ``\"density_matrix\"``: A dense density matrix simulation that can sample measurement outcomes from *noisy* circuits with all measurements at end of the circuit.\n",
+    "\n",
+    "    * ``\"stabilizer\"``: An efficient Clifford stabilizer state simulator that can simulate noisy Clifford circuits if all errors in the noise model are also Clifford errors.\n",
+    "\n",
+    "    * ``\"extended_stabilizer\"``: An approximate simulator for Clifford + T circuits based on a state decomposition into ranked-stabilizer state. The number of terms grows with the number of non-Clifford (T) gates.\n",
+    "\n",
+    "    * ``\"matrix_product_state\"``: A tensor-network statevector simulator that uses a Matrix Product State (MPS) representation for the state. This can be done either with or without truncation of the MPS bond dimensions depending on the simulator options. The default behavior is no truncation.\n",
+    "\n",
+    "    * ``\"unitary\"``: A dense unitary matrix simulation of an ideal circuit. This simulates the unitary matrix of the circuit itself rather than the evolution of an initial quantum state. This method can only simulate gates; it does not support measurement, reset, or noise.\n",
+    "\n",
+    "    * ``\"superop\"``: A dense superoperator matrix simulation of an ideal or noisy circuit. This simulates the superoperator matrix of the circuit itself rather than the evolution of an initial quantum state. This method can simulate ideal and noisy gates and reset, but it does not support measurement.\n",
```

Comment:
```suggestion
    "    * ``\"superop\"``: A dense superoperator matrix simulation of an ideal or noisy circuit. This simulates the superoperator matrix of the circuit itself, rather than the evolution of an initial quantum state. This method can simulate ideal and noisy gates and resets, but it does not support measurements.\n",
```